Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -127,21 +129,41 @@ private extension ReceiptRenderer {
         for line in lines {
             summaryContent += "<tr><td>\(line.title) × \(line.quantity)</td><td>\(line.amount) \(parameters.currency.uppercased())</td></tr>"
         }
-        summaryContent += """
-                            <tr>
-                                <td>
-                                    \(Localization.amountPaidSectionTitle)
-                                </td>
-                                <td>
-                                    \(formattedAmount()) \(parameters.currency.uppercased())
-                                </td>
-                            </tr>
-                            """
+        summaryContent += taxesSummaryRow()
+        summaryContent += amountPaidSummaryRow()
         summaryContent += "</table>"
 
         return summaryContent
     }
 
+    private func taxesSummaryRow() -> String {
+        return taxSummaryRow(title: Localization.totalTaxLineDescription, amount: cartTotals.totalTax)
+    }
+
+    private func taxSummaryRow(title: String, amount: String) -> String {
+        guard amount != "0.00" else {
+            return ""
+        }
+        return summaryRow(title: title, amount: amount)
+    }
+
+    private func summaryRow(title: String, amount: String) -> String {
+        """
+            <tr>
+                <td>
+                    \(title)
+                </td>
+                <td>
+                    \(amount) \(parameters.currency.uppercased())
```

Comment:
One of these days we should switch to NSNumberFormatter. That doesn't have to be today. Would you open an issue?

Reply:
For sure.

In this case, I wanted to use `CurrencyFormatter.formatAmount(amount: String, with currency: String)`, but couldn't easily do it because `CurrencyFormatter` is in the main target. Since we have user-displayable strings outside the main UI now, I think it would be useful to pull helper code like that out to a shared library. 

That seemed pretty disruptive for right now, and we could do with some team discussion about exactly what should be pulled out, and how we want to do that.

I've added #5108